widget_driver/lib/src/utils/test_driver_empty_default_implementation.dart

```diff
@@ -0,0 +1,7 @@
+/// Extend this class for TestDriverDefaultValues that you don't want to implement
+abstract class TestDriverEmptyDefaultImplementation {
```

Comment:
This name is quite long. Can we make it shorter?
Also, now we bind this to be used by TestDrivers, but soon with my other pr, you might want to use this outside of drivers.
Like when you use the future "DependencyProvider" and want to pass in a Test-version-with-no-implementation..

So if possible, I would rename this to not include TestDriver.. and if possible make the name shorter..

Think of the mock version used by mochtail.. it is just called Mock.

Reply:
I [renamed TestDriverEmptyDefaultImplementation to EmptyDefault](https://github.com/bmw-tech/widget_driver/pull/108/commits/ae7c252e4f6f4cb3784d6234cc679dc3769dccb0) please let me know if you like this name :)